pkg/dom0-ztools/rootfs/bin/eve

```diff
@@ -6,15 +6,15 @@ CTR_UA_CMD="ctr --namespace eve-user-apps"
 help() {
   cat <<__EOT__
 Welcome to EVE!
-  commands: enter [qube (assumed pillar)] [command (assumed sh)]
-            enter-user-app <qube>
-            exec qube command
+  commands: enter [(assumed pillar)] [command (assumed sh)]
+            enter-user-app <pillar>
+            exec pillar command
             list
             status
-            start <qube> (requires a qube to be in a destroyed state) 
-            pause <qube>
-            resume <qube>
-            destroy <qube>
+            start <pillar> (requires a pillar to be in a destroyed state) 
+            pause <pillar>
+            resume <pillar>
+            destroy <pillar>
```

Comment:
Can we use more general notation here, for example `service` instead of `qube`, not exactly `pillar`? I think we should also use `service` in `enter` line. `requires a pillar to be in a destroyed state` sounds strange, we do not expect now that pillar will be destroyed.
For the line `exec pillar command` we should wrap `pillar` (I recommend to use `service`) in angle brackets as we can use any service here, not exactly pillar.
For line `enter-user-app` we should use something like `app`, not pillar as we internally search in another namespace (eve-user-apps). As I can see we have no `list-user-apps` command, perhaps will nice to add, but in any case `pillar` is not expected in that line.

Reply:
+1 to @giggsoff since pillar is an example and not the term for the concept.
"service" or "module" works for me. "service" ties in with the fact that these are service containers in Linuxkit.
FWIW I **don't** think we should call the term "container" even though it is technically accurate but it risk confusing folks between the linuxkit service containers and the containers deployed on EVE-OS